src/main/java/dev/latvian/mods/kubejs/util/MobEffectUtil.java

```diff
@@ -0,0 +1,46 @@
+package dev.latvian.mods.kubejs.util;
+
+import dev.latvian.mods.kubejs.typings.Info;
+import net.minecraft.core.Holder;
+import net.minecraft.world.effect.MobEffect;
+import net.minecraft.world.effect.MobEffectInstance;
+import org.jetbrains.annotations.Contract;
+import org.jetbrains.annotations.NotNull;
+
+public class MobEffectUtil {
+	@Contract("_ -> new")
+	@Info("Creates a new instance from the old one")
+	public static @NotNull MobEffectInstance of(MobEffectInstance oldInstance) {
+		return new MobEffectInstance(oldInstance)
+	}
+
+    @Contract("_ -> new")
+	@Info("Creates an instance for the given effect")
```

Comment:
This should say what the default values are for duration and such. Its pretty obvious that its making a MEI from a ME, the info annotations should focus on including information that isn't obvious from the method signature.